### PR TITLE
micro-reddit feature-branch

### PR DESCRIPTION
**Pull request by Rayan84**
This rails application demonstrates rails associations, models, active records, and other features. The application has been built using the following technologies:

- Ruby
- Ruby on Rails
- Git 
- CMD
- Rubocop
- Stylelint
- Atom code editor

### CODE REVIEW

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello @Rayan84  🙌

Great work so far.

You need to implement the works below to get the project approved!🛠️

Please check my points here: 👇

`[REQUIRED]`

 - [x] In `Getting Started` part in README.md, add the following commands, 
        `bundle install` to get the necesary gems.
        `yarn` .
        `rails db:migrate`.

 - [x] In your `Post.rb` you need to validate the title.
 
 -  [x] In your models, you can write validation in one line.


Please make the changes and ask for a re-review.

Feel free to reach me at @lamiasristy 🙋

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  validates :name, presence: true
+  validates :name, length: { in: 3..15 }
+  validates :name, uniqueness: true
```
Comment (LamiaSristy) [inline app/models/user.rb]: You can write like this : 
`validates :name, presence: true, uniqueness: true, length: { in: 3..15 }`

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  validates :body, presence: true
+  validates :body, length: { minimum: 4 }
```
Comment (LamiaSristy) [inline app/models/post.rb]: You can write like this :
`validates :body, presence: true,  length: { minimum: 4  }`

File: app/models/comment.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :body, presence: true
+  validates :body, length: { minimum: 3 }
```
Comment (LamiaSristy) [inline app/models/comment.rb]: You can write like this :
`validates :body, presence: true, length: { minimum: 3 }`

Comment (Rayan84) [reply app/models/comment.rb]: Changes have been applied

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:


_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request merged**